example_cases/MachineParams.py

```diff
@@ -0,0 +1,334 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import xarray as xr
+import cfspopcon
+from scipy import constants  # type: ignore
+import pandas as pd
+
+from cfspopcon.unit_handling import ureg, wraps_ufunc
+from cfspopcon.unit_handling import Quantity, convert_to_default_units, ureg
+from cfspopcon.formulas import calc_plasma_current
+
+# SPARCV1D
+RSPARCV1 = Quantity(1.845, ureg.m)
+aSPARCV1 = Quantity(0.565, ureg.m)
+deltaSPARCV1 = Quantity(0.55, ureg.dimensionless)
+kappaSPARCV1 = Quantity(1.7, ureg.dimensionless)
+BSPARCV1 = Quantity(12.3, ureg.T)
+q95SPARCV1 = Quantity(3.05, ureg.dimensionless)
+# tFlatSPARCV1 = 10
```

Comment:
There are quite a few commented out lines of code in this file, could these be removed?